main.py

```diff
@@ -92,12 +103,10 @@ def create_transaction(self) -> None:
         #     return
 
     def block_creation(self):
-        # WIP: get every known peer, not only the ones im connected to
-        peers = self.get_peers()
-        peers.append(self.my_peer)
-        selectedPeer = random.choice(peers)
+        random.seed(len(self.blocks))
```

Comment:
works the same for all peers only if everyone have the same amount of blocks - are we sure in it? 
As I know it's enough to set the seed only once in the beginning - from docs:

> The seed() method is used to initialize the random number generator.
> The random number generator needs a number to start with (a seed value), to be able to generate a random number.

So I think setting the same seed for all the peers in the beginning is enough and it's important - we should create this random object only once (in the constructor). And it can be some constant value like 21 for example. And in this case we can be sure that all the peers are running block_creation at the same time in parallel and we don't care if all of them are having the same amount of blocks or not

Reply:
sure, Im gonna move it to the constructor and use 21 haha
The only thing is that we don't "fix" it completely because if some peer has different amount of block it will give a different choice, so in the end we will have the same problem if the peers are not in sync